molecule/shared/stable.ver

```diff
@@ -1 +1 @@
-1.2.2
+1.4.0rc1
```

Comment:
Looking at https://github.com/freedomofpress/securedrop/pull/5061/files it looks like this version is also expected to describe the current production release, not the next one.